### ISSUE TITLE
"Reset Virtual Machine" (F11+R) causes boot loop/freeze with EMM386.EXE active (real MS-DOS 6.22)

### ISSUE THREAD

**Issue by DOSFANHK91** [joncampbell123/dosbox-x#3520]
### Describe the bug

When booting an HDD image with real MS-DOS (6.22) installed: 

Regardless of the DOSBOX-X configuration, if EMM386.EXE is being loaded via the CONFIG.SYS file it causes either a boot loop or a freeze when the virtual machine (or the DOS) is being reset via the F11+R (/F11+B) command. 

Forcing a reset via CTRL+ALT+DEL (Sendkey F11+DEL) works totally fine, though. To resolve the issue I have to quit DOSBOX-X entirely via the CRTL+F9 command and start it again.

### Steps to reproduce the behaviour

1. Boot from an HDD image with an MS-DOS 6.22 installation
2. Load the standard EMM386.EXE via the CONFIG.SYS file
3. Reset the virtual machine by pressing F11+R (might have to try it at least two times)

### Expected behavior

F11+R working just like the CTRL+ALT+DEL command.

### What operating system(s) this bug have occurred on?

Windows 10 64bit

### What version(s) of DOSBox-X have this bug?

0.83.25 Windows, 64bit, SDL2 (same behaviour with SDL1)

### Used configuration

```ini
Standard
```

### Have you checked that no similar bug report(s) exist?

- [X] I have searched and didn't find any similar bug report.

### Code of Conduct & Contributing Guidelines

- [X] I agree to follow the code of conduct and the contributing guidelines.

**Comment by maron2000** [comment]
I tried PC-DOS 6.3(US) and 7.0J/V but both can reset by F11 + R.
I tried on 0.83.25 Windows x64 SDL1. 
I doubt it but may be typical to MS-DOS?

**Comment by DOSFANHK91** [comment]
Strange... Does anyone else experience this problem? Maybe I'm the only one :-D

**Comment by maron2000** [comment]
To be clear, the procedures I've done is
imgmount c -> boot c: -> F11+R (all manually)
DOSbox-x returns to the initial screen.
If I do the second time, it crashed. Is this what you're mentioned?

**Comment by DOSFANHK91** [comment]
Yes i think so. It definitely seems to happen (at least) on the second reboot. So far I've tried all kinds of configs and versions but it all comes down to having EMM386.EXE loaded. I know this isn't 100% necessary for running games and programs if you load newer, smaller drivers for cd and mouse but it would still be great to have the system behave "normally" in that regard. :-)
